package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@turbopuffer/turbopuffer",
-  "version": "0.5.2",
+  "version": "0.5.3",
```

Comment:
@morgangallant: This the right place to bump the version?

Reply:
Yep!